polyfill/lib/ecmascript.mjs

```diff
@@ -161,13 +161,17 @@ export const ES = ObjectAssign({}, ES2019, {
   },
   ParseTemporalTimeZoneString: (stringIdent) => {
     try {
-      const canonicalIdent = ES.GetCanonicalTimeZoneIdentifier(stringIdent);
```

Comment:
Is this change necessary? It seems like just removing the `= 'UTC'` ought to do.

Reply:
If there's no time zone, then user gets an "Invalid ISO 8601 string" error message because the fall-through case tries to parse the param as an ISO string. This error message is confusing because I was trying to get a time zone, not parse an ISO string. (As a result of making this PR I learned that you can also pass an ISO string to this method, but most users probably won't know that.)  

I considered reversing the order so the fall through case would return a good error message, but it seemed weird to run ISO parsing code (that almost all the time wouldn't succeed) just to get the error message from the timezone parser. So I wrapped the whole thing. Another approach would be to just wrap the fall-through case in an exception handler call instead of the whole method. What do you think?

Reply:
> Another approach would be to just wrap the fall-through case in an exception handler call instead of the whole method.

Looking at the code again, I think it's clearer (and less scope of change) to just put the exception handler around the ISO parsing call. Updated in latest checkin this morning. 

Let me know if you think this is good or should be removed completely.

Reply:
Yes, that definitely makes all the difference to how the patch reads! Thanks.